src/controller.py

```diff
@@ -224,8 +228,11 @@ def colorize_model(self):
         self.save_settings("vip")
 
         s = sett()
+        shutil.copyfile(s.slicing.stl_file, s.slicing.copy_stl_file)
```

Comment:
Maybe we should wrap it under try/except because it would be sad if something crashes(